### PR TITLE
Corrected my mistake

### PR DESCRIPTION
**Pull request by GabrielRavier**
Hey Clownacy I corrected my mistake. Can I get my merge now ?

### CODE REVIEW

File: s2.asm [hunk 1]
```diff
@@ -350,9 +398,9 @@ GameClrRAM:
 ; loc_394:
 MainGameLoop:
 	move.b	(Game_Mode).w,d0
-	andi.w	#$3C,d0
-	jsr	GameModesArray(pc,d0.w)
-	bra.s	MainGameLoop
+	andi.w	#$3C,d0	; limit Game Mode value to $3C max (remove to add more game modes)
```
Comment (Clownacy) [inline s2.asm]: IIRC, the highest bit is set on the title screen, so removing the instruction outright should cause issues.

Comment (flamewing) [reply s2.asm]: High bit is set during the titlecard, and cleared after; this code is not reachable with high bit set under normal circumstances. You could get there if the game crashed at the titlecard, though.

A better note would be to change it to $7C to allow more game modes.

Comment (GabrielRavier) [reply s2.asm]: I'll do that

**Comment by GabrielRavier** [comment]
I corrected my mistake.

**Comment by GabrielRavier** [comment]
Waiting...

**Comment by flamewing** [comment]
Please don't make useless comments saying you are waiting; it does not help make it be processed any faster, and may actually generate ill will towards it: at least in my case, this kind of comment makes me want to ignore the pull request instead of taking time out of my limited free time to look at it.

**Pull request merged**